### ISSUE TITLE
Mint update - May 07 2022

### ISSUE THREAD

**Issue by Nyran-2004** [pkmnct/robinhood-mint-sync-chrome#64]
Hello,

It appears Mint pushed through an update to their interface and the Robinhood Mint Sync is not automatically initiating any longer.  I was unsuccessful when trying the "Perform Sync Now" or "Trigger Initial Setup" direction from chrome-extension://mogflmdandlpjobbddhopcggkjoggpdo/html/welcome.html as well.  I made sure I was logged into both Mint and Robinhood, trying from Chrome like usual. 

Is anyone else seeing this issue?  Thanks for everyone working on this plugin, it's amazing!

**Comment by liamrosen** [comment]
Yes, can confirm that the new Mint interface has broken the extension. 

@pkmnct, any plans to update?

Thanks for all your hard work!

**Comment by pkmnct** [comment]
@liamrosen Now that Robinhood has native support within Mint, I have not prioritized maintenance of this extension. Is there any specific reason you are using the extension over the native integration?

**Comment by liamrosen** [comment]
No kidding, I had no idea that Robinhood was supported natively now. Thanks!

**Comment by superpowered** [comment]
@pkmnct Could probably update the readme as I assume a bunch of people probably aren't aware of the native support - or that this project is no longer maintained. (at least until RobinHood decides flip flop again)
